### DISCUSSION TITLE
Q&A

### DISCUSSION THREAD

**Discussion by Felixmil** [ColinFay/ur-first-5k#167] [Q&A]
Any comments on running with empty stomach ? I noticed that it is easier for me to run before lunch or breakfast. 
Is it up to personnal preference or is there a "good way" ?

**Comment by ColinFay** [comment]
Running on an empty stomach is advised in some training plans, notably when: 

- you train for longer distances, as you train your body to make an effort without / with a low level of carbohydrates and sugars in your body (which is something that happens when you start running marathons and more)
- you run to lose weight, as your body will go straight to your fat instead of burning your carbohydrates & sugars first 

That being said, that's also a matter of comfort/personal preference. 
For example, I'm not really a morning person, so I am very under-performing whenever I try to go for a run before breakfast :) 
My partner on the other hand prefers going for a run before breakfast.

The only drawback with empty stomach (notably in the morning before breakfast) is that your run might require more energy. 
I suppose that given the length of the current training, there is no real issue with doing it that way if that's what makes you feel better :) 
But if ever you're going for a run in the morning, I'd suggest being sure to at least drink something before so that you're sure to be well hydrated.

**Reply by Felixmil** [reply]
Freshly squizzed orange juice it is then ! 🍊
